articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -0,0 +1,159 @@
+>
+> 1. Go to the [Azure Portal - Application Registration [Prod]](https://portal.azure.com/signin/index/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/JavascriptSpaQuickstartPage/sourceType/docs)
+> 1. Enter a name for your application and click **Register**
+> 1. Follow the instructions to download and configure your new application, adding **http<span/>://localhost:30662/** as Redirect URL and setting the **Implict Flow** automatically for you.
+> [!div renderon="docs"]
+> ### Option 2: Register your application and configure sample [Manual]
+> #### Step 1: Register your application
+> 
+> 1. To register an application, go to the [Azure Portal - Application Registration [Prod]](https://aka.ms/registeredappsprod) and select **New registration**
+> 1. Enter a name for your application, add **http<span/>://localhost:30662/** in Reply URL, and click **Register**
+> 1. Select **Authentication** menu and set **ID tokens** under *Implict Grant* and select 'Save'
+
+> [!div class="sxs-lookup" renderon="portal"]
+> ### Step 1: Configure your application in Azure Portal
+> For the code sample for this quickstart to work, you need to add a reply URL as **http<span/>://localhost:30662/** and enable **Implict Grant**.
+> > [!div renderon="portal" id="makechanges" class="nextstepaction"]
+> > [Make these changes for me]()
+>
+> > [!div id="appconfigured" class="hidden"]
+> > ![Already configured](media/active-directory-windesktop/checkmark.png) Your application is configured with these attributes
+
+> [!NOTE]
+> If you use Visual Studio, the Redirect URL will be set to to *http:<span/>//localhost:30662/* as it is configured in the code sample's project. If you use Python or any other web server, set redirect URL to *http://<span/>localhost:30662/*, run the following command line:
+>> ```bash
+>> python -m http.server 30662
+>> ```
+
+### Step 2: Download your web server or project
+
+- [Download the Visual Studio project](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/VisualStudio.zip)
+- [Download the core project files - for a local web server, such as Python](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/core.zip)
+
+### Step 3: Configure your JavaScript SPA
+
+> [!div renderon="docs"]
+> Edit `msalconfig.js` and replace `Enter_the_Application_Id_here` with the Application ID your application:
+
+> [!div class="sxs-lookup" renderon="portal"]
+> Edit `msalconfig.js` and replace msalconfig with:
+
+```javascript
+var msalconfig = {
+    clientID: "Enter_the_Application_Id_here",
+    redirectUri: location.origin
+};
+```
+
+## More Information
+
+### *msal.js*
+
+MSAL is the library used to sign in users and request tokens used to access an API protected by Microsoft Azure Active Directory. The Quickstart's *index.html* contains reference to the library:
```

Comment:
can we also mention about npmjs? That way user will know that it's available on both CDN and npmjs even though the samples uses it from CDN .

Reply:
@nehaagrawal - will add it

Reply:
@nehaagrawal - can you check below?